changelog.d/6748.misc

```diff
@@ -0,0 +1 @@
+Propagate cache invalidates from workers to other workers.
```

Comment:
does this fix an actual bug we can cite here?

Reply:
I don't think so, it was all best effort anyway. Annoyingly this doesn't automatically fix the destination retry cache stuff as we don't stream invalidations for that currently